generator/views/funnel_analysis_view.py

```diff
@@ -166,8 +166,13 @@ def _event_types_lookml(self) -> List[Dict[str, Any]]:
                               mozfun.event_analysis.aggregate_match_strings(
                                 ARRAY_AGG(
                                   DISTINCT CONCAT(
-                                    {{% if _filters['property_name'] or _filters['property_value'] -%}}
-                                    COALESCE(mozfun.event_analysis.escape_metachars(property_value.value), ''),
+                                    {{% if _filters['property_value'] -%}}
+                                      mozfun.event_analysis.event_property_value_to_match_string(
+                                        properties.index,
+                                        property_value.value
+                                      ),
+                                    {{% elif _filters['property_name'] -%}}
+                                      mozfun.event_analysis.event_property_to_match_string(properties.index),
```

Comment:
This isn't strictly necessary to fix the bug, but since I was already refactoring this logic it seemed reasonable to make this optimization, where if they're only filtering on the property being present use a single regular expression that matches any valid property value rather than having a (potentially huge) regular expression that specifically matches _every_ possible property value.